openzwavemqtt/const.py

```diff
@@ -33,6 +33,34 @@
 EMPTY_PAYLOAD: dict = {}
 
 
+class OpenZWaveStatus(Enum):
+    """Enum with all Status strings for the OZW Daemon."""
+
+    # https://github.com/OpenZWave/qt-openzwave/blob/master/docs/MQTT.md#mqtt-client-status
+
+    OFFLINE = "Offline"
+    STARTING = "starting"
+    STARTED = "started"
+    STOPPED = "stopped"
```

Comment:
In the docs it says `stoped`. Is that just a typo in the docs or is it the api that has a typo?

Reply:
I assumed the api

Reply:
So then we should change to `"stoped"`?

Reply:
No, stoped was the typo. API is stopped.